api/src/pcapi/workers/apps_flyer_job.py

```diff
@@ -44,6 +44,7 @@ def log_user_becomes_beneficiary_event_job(user_id: int) -> None:
         raise AppsFlyerMissingError("user has no apps flyer information")
 
     user_apps_flyer = user.externalIds["apps_flyer"]
+    user_firebase_pseudo_id = user.externalIds["firebase_pseudo_id"] if "firebase_pseudo_id" in user.externalIds else ""
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```suggestion
    user_firebase_pseudo_id = user.externalIds.get("firebase_pseudo_id", "")
```
Fait la même chose